core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/HoltLinearModel.java

```diff
@@ -55,6 +59,29 @@ public HoltLinearModel(double alpha, double beta) {
         this.beta = beta;
     }
 
+    @Override
+    public boolean canBeMinimized() {
+        return true;
+    }
+
+    @Override
+    public MovAvgModel neighboringModel() {
+        double newValue = Math.random();
+        switch ((int) (Math.random() * 2)) {
+            case 0:
+                return new HoltLinearModel(newValue, this.beta);
+            case 1:
+                return new HoltLinearModel(this.alpha, newValue);
+            default:
+                return new HoltLinearModel(newValue, this.beta);    // This should never technically happen...
```

Comment:
worth adding an assert(false) here (maybe with a message) so we can assure it never happens?